PyInstaller/building/makespec.py

```diff
@@ -695,6 +701,7 @@ def main(
         # On Mac OS, the default icon has to be copied into the .app bundle.
         # The the text value 'None' means - use default icon.
         icon_file = 'None'
+    exe_options += "\n    contents_directory='%s'," % (contents_directory or "_internal")
```

Comment:
I think it would make more sense to add `contents_directory` to arguments in spec only if user explicitly specifies it. 

Otherwise, leave it unspecified, and the default argument handling in EXE should take care of it. This way, the default value is specified only there.

Reply:
I concur.

Reply:
> That's the opposite of how we deal with all the other options. Normally the default is in the template so that people editing spec files can see what's changeable.

I'm under impression that `exe_options` is used for "advanced" options that do not have default in the template (`version_file`, `icon`, `uac_admin`, `uac_uiaccess`, `manifest`).

Reply:
The options with defaults set are in the [`templates.py`](https://github.com/pyinstaller/pyinstaller/blob/develop/PyInstaller/building/templates.py).

Reply:
Ah, you're right. Some `EXE()` options are hidden by default.